### ISSUE TITLE
Mouse -> Touch re-wording

### ISSUE THREAD

**Issue by popey** [pseuudonym404/neverball-touch#5]
Some of the texts should probably be made more generic for a mobile device.

"Move the mouse to tilt the floor" -> "Angle the device to tilt the floor"
"Click to begin" -> "Tap to begin"

Note though that it's possible in the near future that someone may run this on a device which _can_ have a keyboard and mouse attached, and they may still want to use the mouse to control it.

**Comment by laurie-bradshaw** [comment]
I changed all the texts in the code that referred to the keyboard and mouse (which need new translations now, which I haven't done), but missed the ones baked into the maps. It's trivial to rebuild the maps with the texts changed in English, but I'm still undecided about translations.

Do you think it would be a good idea to use google translate so there's at least placeholder texts in the po files?

**Comment by laurie-bradshaw** [comment]
Also, I don't think I broke mouse input, I just added sdl finger events, I did change a few hardcoded mouse downs to mouse ups, but that should be fine.

The display code still needs work too, the orientation is hardcoded for portrait screens, needs a check for width > height for default setup, and ideally a config option to select any orientation. Would be nice to show more items on each screen for larger displays too.

edit: I did infact break mouse input in the main game to provide the touchscreen camera rotation
